advanced/is-pattern-1/templates/is/wso2is-pattern-1-identity-server-conf.yaml

```diff
@@ -42,32 +42,26 @@ data:
     [user_store]
     type = "database_unique_id"
 
-    [database.user]
-    url = "jdbc:mysql://wso2is-mysql-db-service:3306/WSO2IS_SHARED_DB?autoReconnect=true&amp;useSSL=false"
-    username = "wso2carbon"
-    password = "wso2carbon"
-    driver = "com.mysql.cj.jdbc.Driver"
-
-    [database.user.pool_options]
-    validationQuery = "SELECT 1"
-
-    [transport.https.properties]
-    proxyPort="443"
-
     [database.identity_db]
+    type = "mysql"
     url = "jdbc:mysql://wso2is-mysql-db-service:3306/WSO2IS_IDENTITY_DB?autoReconnect=true&amp;useSSL=false"
     username = "wso2carbon"
     password = "wso2carbon"
     driver = "com.mysql.cj.jdbc.Driver"
-
     [database.identity_db.pool_options]
     validationQuery = "SELECT 1"
 
     [database.shared_db]
+    type = "mysql"
```

Comment:
Is this required when url is given?

Reply:
Followed the 6.0.0 changes.
https://github.com/wso2/kubernetes-is/blob/6.0.x/advanced/is-pattern-1/templates/is/wso2is-pattern-1-identity-server-conf.yaml